selfdrive/car/gm/carcontroller.py

```diff
@@ -42,23 +42,21 @@ def update(self, CC, CS):
     can_sends = []
 
     # Steering (50Hz)
-    # Avoid GM EPS faults when transmitting messages too close together: skip this transmit if we just received the
-    # next Panda loopback confirmation in the current CS frame.
-    if CS.lka_steering_cmd_counter != self.lka_steering_cmd_counter_last:
-      self.lka_steering_cmd_counter_last = CS.lka_steering_cmd_counter
```

Comment:
What this used to assume was that one message missing on the loopback bus was due to lag, and two+ messages blocked. We however don't know either for sure without a blocked/192 bus. If we block we want to re-transmit, and if we lag we want to increment counter as normally, but skip sending.

Perhaps we should do the panda block recovery with this